README.md

```diff
@@ -151,6 +168,24 @@ set_namespace("MyApplication")
 
 Flushes the current MetricsContext to the configured sink and resets all properties, dimensions and metric values. The namespace and default dimensions will be preserved across flushes.
```

Comment:
I think this should match the Java README. Either state that custom dimensions are reset based on `flush_preserve_dimensions` or only write that properties and metric values are reset.

Reply:
Makes sense!